### ISSUE TITLE
Loss for fine-tuning metric depth models

### ISSUE THREAD

**Issue by pestrstr** [LiheYoung/Depth-Anything#78]
Hi @LiheYoung, thank you for your work and for open-sourcing it!
I have a question about the loss that you used for fine-tuning your Depth-Anything encoder on metric-depth estimation.
From the supplementary material of your paper (Section 6), it is reported that you used the original ZoeDepth repo, replacing the MiDaS encoder and tuning a few hyper-parameters. 
Did these hyper-parameters affect the computation of your loss? In particularly, did you rely on the scale-invariant log loss (+ gradient matching term like in ZoeDepth and DINOv2 repos) even when fine-tuning on metric depth to obtain values in an absolute scale? 

Thank you in advance!

**Comment by LiheYoung** [comment]
Hi, we did not change the loss function of ZoeDepth. The "a few hyper-parameters" only include the image resolution (our patch size is 14, while ZoeDepth is 16) and the learning rate.

**Comment by pestrstr** [comment]
Got it, thank you!
